che-incubator/chectl
# feat: Detect operator subscription name

Signed-off-by: Mykola Morhun <mmorhun@redhat.com>

### What does this PR do?
Detects Che Operator subscription name instead of using the hardcoded one.


### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/20424

### How to test this PR?
1. Create a subscription to Che operator with an arbitrary name, for example:
```yaml
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: my-test
  namespace: eclipse-che
spec:
  channel: stable
  installPlanApproval: Manual
  name: eclipse-che
  source: test
  sourceNamespace: openshift-marketplace
  startingCSV: eclipse-che-preview-openshift.v7.36.1-all-namespaces
```
2. Run `server:delete` command and check that the subscription was found and deleted.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
